notebooks/svo-ranking/02_ranking_suggestions.py

```diff
@@ -3,39 +3,36 @@
 import os
 import pandas as pd
 
+DIR_RAW = os.getenv('DIR_DATA_RAW', 'data/raw')
+DIR_PROCESSED = os.getenv('DIR_DATA_PROCESSED', 'data/processed')
 
-DIR_RAW = os.getenv('DIR_DATA_RAW')
-DIR_PROCESSED = os.getenv('DIR_DATA_PROCESSED')
-
-# get GA and content store data
+# get GA data
 df_ga = pd.read_csv(filepath_or_buffer=DIR_RAW + '/bq_results_svo_ranking.csv')
```

Comment:
this query processes 5TB of data?!?!? 
Please can we use something like:

`  SELECT
    COUNT(*) as n,
    ANY_VALUE(hits.page.pageTitle),
    (SELECT value FROM hits.customDimensions WHERE index=4) AS content_id,
  FROM
    `govuk-bigquery-analytics.87773428.ga_sessions_intraday_*` AS sessions
  CROSS JOIN
     UNNEST(sessions.hits) AS hits
  WHERE
    _TABLE_SUFFIX = FORMAT_DATE('%Y%m%d',(CURRENT_DATE()))
  and hits.type = 'PAGE'
  GROUP BY content_id
  `